### PR TITLE
[WIP] Giftcode command

### PR DESCRIPTION
**Pull request by Doges**
wip

### CODE REVIEW

File: src/commands/bot/giftcode/generate.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    this.parameters = new CommandParameters(this,
+      new NumberParameter(({ min: 25000, max: 50000, missingError: 'commands:giftcode.subcommands.generate.noValue' }))
+    )
+
+    this.requirements = new CommandRequirements(this, { guildOnly: true, databaseOnly: true, openDms: true, onlyOldAccounts: true })
+  }
+
+  async run ({ t, author, channel }, value) {
+    const embed = new SwitchbladeEmbed(author)
+    const Uembed = new SwitchbladeEmbed(author)
```
Comment (perronosaurio) [inline src/commands/bot/giftcode/generate.js]: that is unnecesary

Comment (Doges) [reply src/commands/bot/giftcode/generate.js]: thats necessary

Comment (Doges) [reply src/commands/bot/giftcode/generate.js]: like, its 2 embeds, one sends to author and the another one sends to the chat

Comment (pedrofracassi) [reply src/commands/bot/giftcode/generate.js]: it is unnecessary. you don't need to declare it, just put new SwitchbladeEmbed inside .send

Comment (Doges) [reply src/commands/bot/giftcode/generate.js]: the uembed is actually more easily, and i just copied from the last pr

**Comment by Doges** [comment]
Remaking this later.

**Pull request closed without merging**